adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -354,21 +353,21 @@ public void run() {
                 if (accountList != null) {
                     for (Account targetAccount : accountList) {
                         Logger.v(TAG, "remove tokens for:" + targetAccount.name);
-                        if (targetAccount != null) {
```

Comment:
the way foreach works in java, the loop will break if null

---
In reply to: [65639679](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65639679) [](ancestors = 65639679)